app/Http/Controllers/CompetitionsController.php

```diff
@@ -104,9 +105,9 @@ public function destroy(Competition $competition)
      * @param  \Gladiator\Models\Competition  $competition
      * @return \League\Csv\ $csv
      */
-    public function export(Competition $competition)
+    public function export(Competition $competition, Request $request)
     {
-        $csv = $this->manager->exportCSV($competition, true);
+        $csv = $this->manager->exportCSV($competition, $request->input('hasReportback'));
```

Comment:
we can if you think it looks better, but functionally it doesnt make a difference